recipe/build_nodepTrue.sh

```diff
@@ -22,5 +22,9 @@ mkdir -p ${PACKAGE_HOME} ${PREFIX}/bin
 mv * ${PACKAGE_HOME}/
 ln -s ${PACKAGE_HOME}/git-annex ${PREFIX}/bin/
 ln -s ${PACKAGE_HOME}/git-annex-shell ${PREFIX}/bin/
+ln -s ${PACKAGE_HOME}/git-annex-webapp ${PREFIX}/bin/
+ln -s ${PACKAGE_HOME}/git-remote-annex ${PREFIX}/bin/
+ln -s ${PACKAGE_HOME}/git-remote-p2p-annex ${PREFIX}/bin/
+ln -s ${PACKAGE_HOME}/git-remote-tor-annex ${PREFIX}/bin/
```

Comment:
given all that shouldn't we replace all those with one above with

```suggestion
ln -s ${PACKAGE_HOME}/git-annex-* ${PREFIX}/bin/
```
?